db/seeds.rb

```diff
@@ -2,7 +2,7 @@
 BootstrapUser.(ihid)
 
 # Seed tracks
-Git::SeedsTracks.seed!
+Git::SeedTracks.call
```

Comment:
Can you change this to `Git::SeedTracks.()` pls :)

Reply:
Done